pkg/controller/controller_binding.go

```diff
@@ -158,18 +162,20 @@ func (c *controller) reconcileServiceBinding(binding *v1beta1.ServiceBinding) er
 		return err
 	}
 
-	glog.V(4).Infof("Processing ServiceBinding %v/%v", binding.Namespace, binding.Name)
+	glog.V(4).Infof(
+		`%s: Processing ServiceBinding %s/%s`,
+		typez, binding.Namespace, binding.Name,
+	)
 
 	instance, err := c.instanceLister.ServiceInstances(binding.Namespace).Get(binding.Spec.ServiceInstanceRef.Name)
 	if err != nil {
-		s := fmt.Sprintf("ServiceBinding \"%s/%s\" references a non-existent ServiceInstance \"%s/%s\"", binding.Namespace, binding.Name, binding.Namespace, binding.Spec.ServiceInstanceRef.Name)
+		s := fmt.Sprintf(
```

Comment:
Since this string is just getting tacked onto conditions and events that intrinsically reference the resource, there's no need for it to contain the resource name and type

Reply:
This happens a few times in this file. I am going to try to just log the new format, not attach it to the object